circuit-types/src/balance.rs

```diff
@@ -5,7 +5,7 @@ use std::ops::Add;
 
 use circuit_macros::circuit_type;
 use constants::{AuthenticatedScalar, Scalar, ScalarField};
-use mpc_relation::{ConstraintSystem, Variable};
+use mpc_relation::{traits::Circuit, Variable};
```

Comment:
getting unresolved import errors for `traits`, is it a public module?

Reply:
yes, it's just dependent on the other PR